sniproxy.cpp

```diff
@@ -311,18 +305,38 @@ void SNIProxy::sendClick(uint8_t mouseButton, int x, int y)
     //note x,y are not actually where the mouse is, but the plasmoid
     //ideally we should make this match the plasmoid hit area
 
-    qCDebug(SNIPROXY) << "Sending click " << mouseButton << "to" << x << y;
+    qCDebug(SNIPROXY) << "Received click" << mouseButton << "with passed x*y" << x << y;
 
     auto c = QX11Info::connection();
 
-    //set our window so the middle is where the mouse is
+    auto cookie = xcb_query_pointer(c, m_windowId);
+    QScopedPointer<xcb_query_pointer_reply_t> pointer(xcb_query_pointer_reply(c, cookie, Q_NULLPTR));
```

Comment:
it will, but the mouse will always be out of the geometry of x and y.

For keyboard activation:

Old version: menu won't always appear, but if it does it'll be in the right place
your version: menu should always appear, but will always be in wrong place

Lets go with yours